cmd/createtree/main.go

```diff
@@ -151,24 +152,30 @@ func newPK(opts *createOpts) (*any.Any, error) {
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PKCS11ConfigFile":
+		if opts.pkcs11ConfigPath == "" {
+			return nil, errors.New("empty PKCS11 config file path")
+		}
+		return ptypes.MarshalAny(&keyspb.PKCS11ConfigFile{opts.pkcs11ConfigPath})
```

Comment:
The module is likely to be fixed for a particular deployment of Trillian, right? But the config would be different for each tree, to specify different keys, correct?

If my assumptions above hold, then I'd recommend making the module path controlled by a flag, and make the proto here contain everything else (i.e. no file paths in the proto).